packages/adena-extension/src/pages/certify/reveal-private-phrase/index.tsx

```diff
@@ -12,17 +12,11 @@ export const RevealPrivatePhrase = () => {
   const navigate = useNavigate();
   const { wallet } = useWalletContext();
   const [showBlurScreen, setShowBlurScreen] = useState(true);
-  const [seeds, setSeeds] = useState<Array<string>>([]);
 
-  useEffect(() => {
-    init();
-  }, []);
-
-  const init = async () => {
+  const seeds = useMemo(() => {
     const mnemonic = wallet?.mnemonic || '';
-    const seeds = `${mnemonic}`.split(' ');
-    setSeeds(seeds);
-  };
+    return `${mnemonic}`.split(' ');
```

Comment:
I think it's same to `return mnemonic.split(' ');`

Reply:
I agree. Thanks for the feedback.